README.md

```diff
@@ -44,14 +44,20 @@ Third-party A-Frame components (of the entity-component system).
 
 - [aframe-layout](https://github.com/ngokevin/aframe-layout)
 
-### React
+### Adapters and Plugins for Frameworks and Libraries 
```

Comment:
hmm a bit lengthy. maybe just `With Frameworks and Libraries`?

then move webpack into it as well